packages/lesmis/src/constants/constants.js

```diff
@@ -15,53 +15,62 @@ export const LESMIS_PERMISSION_GROUPS = {
   ADMIN: 'LESMIS Admin',
 };
 
+// Dashboard Tab View Names
+export const DASHBOARD_REPORT_TAB_VIEW = 'DashboardReportTabView';
+export const FAVOURITE_DASHBOARD_TAB_VIEW = 'FavouriteDashboardTabView';
+export const TAB_TEMPLATE = 'TabTemplate';
+
 // Dashboard Constants
-export const SUB_DASHBOARD_OPTIONS = [
+export const DROPDOWN_OPTIONS = [
   {
-    code: 'profile',
+    value: 'profile',
+    tabViewType: DASHBOARD_REPORT_TAB_VIEW,
     useYearSelector: true,
     exportToPDF: true,
   },
   {
-    code: 'indicators',
-    label: 'dashboards.freeIndicatorSelection',
+    value: 'favourites',
```

Comment:
Can we make this a constant? eg.
`const FAVOURITES_DASHBOARD_CODE = 'favourites'`

I see it being re-used in packages/lesmis/src/utils/getExportableSubDashboards.js